airflow/providers/http/example_dags/example_http.py

```diff
@@ -42,6 +42,7 @@
 dag.doc_md = __doc__
 
 # t1, t2 and t3 are examples of tasks created by instantiating operators
+# [START howto_operator_http_t1]
 t1 = SimpleHttpOperator(
     task_id='post_op',
```

Comment:
```suggestion
post_op = SimpleHttpOperator(
    task_id='post_op',
```
I like it when task instance and task_id are inline.WDYT?

Reply:
me too - will change it :)